app/javascript/packages/document-capture/components/acuant-camera.tsx

```diff
@@ -307,6 +334,45 @@ function AcuantCamera({
     },
     [onImageCaptureSuccess],
   );
+  const faceCaptureCallback = {
+    onDetectorInitialized: () => {
+      console.log('onDetectorInitialized');
+      // This callback is triggered when the face detector is ready.
+      // Until then, no actions are executed and the user sees only the camera stream.
+      // You can opt to display an alert before the callback is triggered.
+    },
+    onDetection: (text) => {
+      console.log('onDetection', text);
+      // Triggered when the face does not pass the scan. The UI element
+      // should be updated here to provide guidence to the user
+    },
+    onOpened: () => {
+      // Camera has opened
+      console.log('onOpened');
+    },
+    onClosed: () => {
+      // Camera has closed
+      console.log('onClosed');
+    },
+    onError: (error) => {
+      // Error occurred. Camera permission not granted will
+      // manifest here with 1 as error code. Unexpected errors will have 2 as error code.
+      console.log('onError', error);
+    },
+    onPhotoTaken: () => {
+      // The photo has been taken and it's showing a preview with a button to accept or retake the image.
+      console.log('onPhotoTaken');
+    },
+    onPhotoRetake: () => {
+      // Triggered when retake button is tapped
+      console.log('onPhotoRetake');
+    },
+    onCaptured: (base64Image) => {
+      // Triggered when accept button is tapped
+      console.log('onCaptured');
+      //onImageCaptureSuccess({image: base64Image});
+    },
```

Comment:
This is currently not being called when I click the accept button.
- I'm seeing the `onDetection`, `onOpened`, and `onClosed` callbacks being called.
- I'm getting a console error about a content security policy failure.
- That console error goes away when I comment out the lines that load the `opencv` script (in `context/acuant`)

The mobile setup I use to see these console errors is [here](https://github.com/18F/identity-idp/blob/main/docs/mobile.md#iphone--safari-or-chrome).

Reply:
I don't quite feel stuck yet, but figured I'd ask before that happens. @aduth any thoughts on this?
- It seems possible that I've missed a step in the directions I'm following [here](https://github.com/Acuant/JavascriptWebSDKV11#face-capture-with-real-time-face-detection-and-acuant-passive-liveness)?
- This callback setup is new since we removed the selfi UI code, so no help [there](https://github.com/18F/identity-idp/pull/7113).

Reply:
Could the content security policy error be the root cause? Could be worth trying to disable that temporarily to see if it behaves any different.

Reply:
> (safely)

I wouldn't worry too much about being safe with it as long as it's just for testing locally.

You could probably gut some parts of `config/initializers/content_security_policy.rb` or the `override_csp_to_allow_acuant` in `app/controllers/concerns/idv/acuant_concern.rb` to test.

Reply:
Thanks! Those code pointers were really helpful. No more content security policy error (for testing).

Turned out not to be the problem, but I think I'm on the right track again. I'm also reassured by the callbacks working correctly in our [tiny test app](https://github.com/18F/identity-acuant-test).